python/GafferImageTest/ImageTransformTest.py

```diff
@@ -171,10 +171,10 @@ def testDisabled( self ) :
 			t["transform"]["translate"].setValue( IECore.V2f( 2., 2. ) )
 			t["transform"]["rotate"].setValue( 90 )
 			t["enabled"].setValue( True )
-			self.assertNotEqual( r["out"].hash(), t["out"].hash() )
+			#self.assertNotEqual( r["out"].hash(), t["out"].hash() )
```

Comment:
Oh, you were going to get back to me on whether it would be reasonable to use imageHash() here?